src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -69,8 +84,13 @@ class _StatsWatcher:
     _no_remote_peer: int
     _lock: threading.Lock
     _condition: threading.Condition
+    _metadata_keys: frozenset
```

Comment:
```suggestion
    _metadata_keys: frozenset[str]
```

Reply:
We [can't subscriptable a type](https://btx-internal.corp.google.com/invocations/445d8563-c870-4e0d-804f-1d4e98443a7d/log):
```
==================== Test output for //src/python/grpcio_tests/tests_py3_only/interop:xds_interop_client_test:
Traceback (most recent call last):
  File "/root/.cache/bazel/_bazel_root/954bb7512d44d20015390af6e76121c6/sandbox/processwrapper-sandbox/2694/execroot/com_github_grpc_grpc/bazel-out/k8-fastbuild/bin/src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client_test.runfiles/com_github_grpc_grpc/src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client_test.py", line 27, in 
    import xds_interop_client
  File "/var/local/jenkins/grpc/src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py", line 78, in 
    class _StatsWatcher:
  File "/var/local/jenkins/grpc/src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py", line 87, in _StatsWatcher
    _metadata_keys: frozenset[str]
TypeError: 'type' object is not subscriptable
```

Reply:
Ah. Python 3.8. We should do `typing.DefaultDict` then:
```suggestion
    _metadata_keys: DefaultDict[str]
```

Reply:
Ugh. I meant 
```suggestion
    _metadata_keys: FrozenSet[str]
```

Reply:
Gentle ping here